anvil-python/anvil/commands/haproxy.py

```diff
@@ -186,6 +187,28 @@ def get_unit_timeout(self) -> int:
         return HAPROXY_UNIT_TIMEOUT
 
 
+class UpgradeHAProxyUnitCharms(UpgradeCharm):
+    def __init__(
+        self,
+        client: Client,
+        jhelper: JujuHelper,
+        manifest: Manifest,
+        model: str,
+    ):
+        super().__init__(
+            "Upgrade HAProxy unit charms",
+            "Upgrading HAProxy unit charms.",
```

Comment:
Could you please upgrade all the other classes like this (maas-region, maas-agent, postgresql)?

```suggestion
            "Upgrading HAProxy charm",
```